containers/backend/backend/app/schemas.py

```diff
@@ -113,9 +114,9 @@ class AdaptationVariables(DataVariables):
 
 
 class AdaptationCostBenefitRatioParameters(DataParameters):
-    eael_days: conint(ge=1, le=30)
+    eael_days: Annotated[int, Field(ge=1, le=30)]
 
-    @validator("eael_days")
+    @field_validator("eael_days")
```

Comment:
While we're at it - was the reason for keeping this quirk fix since this was first released because it was too time consuming to recalculate the raw data?

Reply:
Yes - may revisit as @thomas-fred works on easier reproducibility of the risk calculations.